### ISSUE TITLE
Manual deployment guide is missing description of Fargate options for AWS Batch compute environment

### ISSUE THREAD

**Issue by slangenbach** [Netflix/metaflow-docs#6]
Although available through the [Cloud Formation template](https://github.com/Netflix/metaflow-tools/blob/master/aws/cloudformation/metaflow-cfn-template.yml), the manual deployment guide does not mention the **Fargate** options (Fargate, Fargate Spot) when creating compute environments for AWS Batch. Given the excellent quality and level of detail of the guide, this should be added.

**Comment by savingoyal** [comment]
@slangenbach Yes, that's high up on my TODO list :) You should see it appear rather sooner.
